sdk/storage/AzureStorageBlob/Source/StorageBlobClient.swift

```diff
@@ -73,13 +77,11 @@ public class StorageBlobClient: PipelineClient, PagedCollectionDelegate {
 
     // MARK: Initializers
 
-    /**
-     Create a Storage blob data client.
-     - Parameter accountUrl: Base URL for the storage account.
-     - Parameter credential: A credential object used to retrieve authentication tokens.
-     - Parameter withOptions: A `StorageBlobClientOptions` object to control the download.
-     - Returns: A `StorageBlobClient` object.
-     */
+    /// Create a Storage blob data client.
+    /// - Parameters:
+    ///   - accountUrl: Base URL for the storage account.
+    ///   - credential: A credential object used to retrieve authentication tokens.
+    ///   - options: A `StorageBlobClientOptions` object to control the download.
```

Comment:
For docstrings, shouldn't we be naming the parameters by their external name (here, "withOptions")? Not sure what the guidance is on that, I can see arguments either way.

Reply:
I would agree with you... but when you use the "Add Document" functionality in XCode, it exposes the internal names so I went with that.